crates/metadata-store/src/raft/store.rs

```diff
@@ -0,0 +1,326 @@
+// Copyright (c) 2024 - Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
```

Comment:
Note that all those new files have an old header with old dates

Reply:
It was created in 2024 but wasn't merged until 2025. I will correct the license headers.